x-pack/apm-server/sampling/processor_test.go

```diff
@@ -563,6 +579,9 @@ func TestStorageGC(t *testing.T) {
 	require.NoError(t, err)
 	t.Cleanup(func() { badgerDB.Close() })
 	config.DB = badgerDB
+	config.Storage = eventstorage.
```

Comment:
Should this be closed on test cleanup?

Reply:
Thanks, I mistakenly removed it in my earlier commit. I have also moved the processor stop to `Cleanup` function for `BenchmarkProcess` so that it doesn't affect benchmark results (thanks @marclop for the tip).